test/tap/ping.js

```diff
@@ -47,7 +47,7 @@ test('npm ping', function (t) {
       t.ifError(err, 'no error output')
       t.notOk(code, 'exited OK')
 
-      t.same(JSON.parse(stdout), pingResponse)
+      t.same(stdout, 'Ping succeeded: ' + JSON.stringify(pingResponse))
```

Comment:
This message should be `t.same(stdout, 'Ping success: ' + JSON.stringify(pingResponse))`